cmd/upgrade.go

```
@@ -80,7 +80,7 @@ func upgrade(cobraCmd *cobra.Command) {
 	check(err)
 
 	if (sslCert != "" && sslKey == "") || (sslCert == "" && sslKey != "") {
-		log.Warn("Both --sslCert and --sslKey must be provided together. Only one was specified, so no certificate will be used.")
+		check(errors.New("Both --sslCert and --sslKey must be provided together. Only one was specified."))
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not end with punctuation or newlines

(staticcheck)